editorconfig.go

```diff
@@ -50,9 +50,9 @@ const (
 )
 
 var (
-	v0_10_0 = semver.Version{
+	v0_9_0 = semver.Version{
```

Comment:
Is this change intentional?

Reply:
Indeed. The test is done with 0.8.0, rather than 0.9.0. A misunderstanding from my side.

https://github.com/editorconfig/editorconfig-core-test/blob/c87a3019abc182961bd6c4b420c4d534f006541e/properties/CMakeLists.txt#L41-L44